programmers/문자열-내-p와-y의-개수-세기.js

```diff
@@ -0,0 +1,33 @@
+function solution(s){
+    var count_p = 0;
+//s내부의 p의 개수를 세줄 변수를 선언하였습니다.    
+    var search_p = 'p';
+// indexof라는 함수를 이용하여 p를 찾기 위하여 search_p라는 문자 p를 담고있는 변수를 선언하였습니다. 
+    var index_p = s.indexOf(search_p);
+// indexof라는 함수를 사용하여 s내부의 p의 위치를 입력받는 index_p를 선언하였습니다.
+    while(index_p !== -1){
+        count_p ++;
+        index_p = s.indexOf(search_p,index_p+1);
+    }
+/*index_p가 문자열 s의 마지막인덱스 까지도는 동안나오는 p의 개수를 count_p에 누적시키고,
+ indexof함수는 searchvalue가 없게 되면 -1을 리턴하므로 while문을 빠져나와 줍니다.*/
+    var count_y = 0;
+    var search_y = 'y';
+    var index_y = s.indexOf(search_y);
```

Comment:
위에 코드와 마찬가지로 let과 const로 선언해주시면 좋구  변수 선언은  이왕이면  코드 중간 부터가 아니라 제일 
상단에 미리 선언을 해주시는게 좋습니다
```suggestion
   const index_y = s.indexOf('y');
   let count_y = 0;
    
```